testdriver/testdriver.go

```diff
@@ -149,27 +150,13 @@ func NewTestDriver(c *Config) (*TestDriver, error) {
 
 	// We use the minimal srcVimrc, augmented with some testdriver specific
 	// commands.
-	//
-	// Before a test script starts, the files from the txtar archive are
-	// extracted. Then govim (and gopls) are started. In some rare cases, gopls
-	// can end up calculating diagnostics before the testscript has started.
-	// Which means that it can be the case, because govim and Vim are already up
-	// and running, that the first buffer actually gets used for quickfix as
-	// opposed to, say, the first file we open.
-	//
-	// Instead below we copen then cclose in vimrc to reserve and establish a
-	// definite buffer number for the quickfix window. The quickfix buffer will
-	// always be buffer number 2 (vim creates a buffer when opening, copen then
-	// creates a new window and buffer, the original buffer created when opening
-	// vim gets used for the first file we open, say)
 	vimrcBytes, err := ioutil.ReadFile(srcVimrc)
 	if err != nil {
 		return nil, fmt.Errorf("failed to read from src vimrc %v: %v", srcVimrc, err)
 	}
 	vimrc := bytes.NewBuffer(vimrcBytes)
```

Comment:
Optional: skip the buffer since you are no longer augmenting the vimrc.

Reply:
I was actually trying to be forward thinking in leaving in a convenient placeholder for adding augmentations to the vimrc. But it's not really work it so I'll switch to using `copyFile` as elsewhere because that's all we need here.